hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/realtime/HoodieRealtimePath.java

```diff
@@ -100,13 +105,12 @@ public boolean includeBootstrapFilePath() {
     return pathWithBootstrapFileStatus != null;
   }
 
-  public BaseFileWithLogsSplit buildSplit(Path file, long start, long length, String[] hosts) {
-    BaseFileWithLogsSplit bs = new BaseFileWithLogsSplit(file, start, length, hosts);
-    bs.setBelongsToIncrementalQuery(belongsToIncrementalPath);
+  public HoodieRealtimeFileSplit buildSplit(Path file, long start, long length, String[] hosts) {
+    HoodieRealtimeFileSplit bs = new HoodieRealtimeFileSplit(file, start, length, hosts);
+    bs.setBelongsToIncrementalQuery(belongsToIncrementalQuery);
```

Comment:
why pathWithBootstrapFileStatus is not considered in this buildSplit? or don't we need to have another builder ?